securedrop/models.py

```diff
@@ -117,29 +117,13 @@ def collection(self) -> 'List[Union[Submission, Reply]]':
         return collection
 
     @property
-    def fingerprint(self):
+    def fingerprint(self) -> 'Optional[str]':
         return current_app.crypto_util.get_fingerprint(self.filesystem_id)
 
-    @fingerprint.setter
-    def fingerprint(self, value):
-        raise NotImplementedError
-
-    @fingerprint.deleter
-    def fingerprint(self):
-        raise NotImplementedError
```

Comment:
Trying to set or delete a `@property` already raises an exception so this seems unnecessary.

Reply:
OK, but we should still test that `AttributeError` is raised when attempting to set or delete it.

Reply:
Then the test case will be just for "Is this attribute defined as a Python `@property`?" which seems super specific and not very important? I can add it back if needed.

Reply:
Another way to think about it, which I mentioned in my other comment: would you add this same test for every field declared as a `@property` in the securedrop code base?

Reply:
No, but `Source.public_key` was [originally](https://github.com/freedomofpress/securedrop/pull/3619/commits/7adb2544851f36a689a16e3bc63f6358ef05e466) added as a plain property, as you've done, and then it was deemed important to [explicitly prevent and test for writes](https://github.com/freedomofpress/securedrop/pull/3619/commits/1b852b0e3e09550f6e3d7309789735577caafab4). The reasoning was not captured in #3619 but the comments suggest that the change was deliberate. While characterized as a nit, I'm guessing they wanted an extra speed bump in case someone later changed `public_key` again. 

All that said, we can remove the old tests.